src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/MediaSystem.cs

```diff
@@ -79,6 +79,17 @@ public static bool Startup(MediaContext mc)
                 }
                 s_refCount++;
             }
+
+             /// <summary>
+            /// Set renderOption if Hardware acceleration is enabled for RDP
+            // </summary>
+
+            bool isHardwareAccelerationEnabledForRDP = false;
+            AppContext.TryGetSwitch("System.Windows.Media.isHardwareAccelerationEnabledForRDP", out isHardwareAccelerationEnabledForRDP);
```

Comment:
How about use Pascal case as the configuration string?

```suggestion
            AppContext.TryGetSwitch("System.Windows.Media.IsHardwareAccelerationEnabledForRDP", out isHardwareAccelerationEnabledForRDP);
```

I found that other code is using Pascal case as the configuration string.

https://github.com/dotnet/wpf/blob/2804219c720fd90e5cbba9df535eb2ed5c8a6dd1/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridCell.cs#L52

Reply:
@lindexi - Thank you for your review. I have made the required change.